modules/core/src/main/java/org/apache/ignite/Ignition.java

```diff
@@ -517,6 +517,16 @@ public static Ignite ignite(@Nullable String name) throws IgniteIllegalStateExce
     }
 
     /**
+     * Get or start grid instance, if it wasn't started yet.
```

Comment:
Better java doc description.

Gets or starts new grid instance if it hasn't been started yet.